src/rules/fileHeaderRule.ts

```diff
@@ -20,6 +20,7 @@ import * as ts from "typescript";
 import * as Lint from "../index";
 
 const ENFORCE_TRAILING_NEWLINE = "enforce-trailing-newline";
+const ALLOW_MULTILINE = "allow-multiline";
```

Comment:
I don't think this is a good option name. The rule _currently_ supports block comments, which are typically "multiline". So I think this should be called `allow-singleline`

Reply:
Option is called `allowSingleLineComments`.